.gitignore

```diff
@@ -338,6 +341,7 @@ src/extension/package.nls.*.json
 **/build/
 **/package-lock.json
 **/*.vsix
+src/extension/.vscode-test
```

Comment:
I added `src/extension/.vscode-test` & `src/extension/src/template_test` to .gitignore because they both dynamically generate files which don't need to be version controlled, similar to a `/dist` folder